src/main/scala/nl.knaw.dans.easy.stage/dataset/EMD.scala

```diff
@@ -52,6 +58,33 @@ object EMD extends DebugEnhancedLogging {
     }
   }
 
+  def addAgreementFields(emd: EasyMetadata)(implicit s: Settings): Unit = {
+    val agreementPath = s.bagitDir.toPath.resolve(Paths.get("metadata/agreements.xml"))
+    if (Files.exists(agreementPath)) {
+      val agreementsXml = XML.loadFile(agreementPath.toFile)
+      if (BooleanUtils.toBoolean((agreementsXml \\ "depositAgreementAccepted").text)) {
+        emd.getEmdRights.setAcceptedLicense(true)
+      }
+      if (BooleanUtils.toBoolean((agreementsXml \\ "containsPrivacySensitiveData").text)) {
+        emd.getEmdOther.getEasRemarks.add(new BasicRemark("containsPrivacySensitiveData"))
+      }
+    }
+    else {
+      logger.info("agreements.xml not found, not setting agreement data")
+    }
+  }
+
+  private def addMessageForDataManager(emd: EasyMetadata)(implicit s: Settings): Unit = {
+    val msgForDataManager = new File(s.bagitDir, "metadata/message-for-the-datamanager.txt")
+    if (msgForDataManager.exists) {
+      val content = Source.fromFile(msgForDataManager).mkString
+      emd.getEmdOther.getEasRemarks.add(new BasicRemark(content))
+    }
+    else {
+      debug("could not find message for message-for-the-datamanager.txt")
```

Comment:
Why log the absence of `agreements.xml` as `info`, but `message-for-the-datamanager.txt` as `debug`? I'd prefer them both to be on `info` level.

```suggestion
      logger.info("message-for-the-datamanager.txt not found, not setting a remark")
```